docs/docs/workflows/networking/README.md

```diff
@@ -6,7 +6,7 @@ If you use a service that requires you whitelist a range of IP addresses, you ha
 
```

Comment:
I looked at [the rest of this doc](https://pipedream-docs-git-docs-vpc-pipedreamers.vercel.app/docs/workflows/networking/) and we should probably just remove it entirely and redirect to `/workflows/vpc` in the `vercel.json`.

Reply:
I considered that too, but don't we don't to describe that by default all workflows belong in `us-east-1` in a separate doc than VPCs?

This info seems like it should be categorized outside of the VPC documentation, since it describes the default behavior.